Lesson_3/Lesson_3.js

```diff
@@ -0,0 +1,50 @@
+// Задание 1
+function isThe(num) {
+    if (num<2) {
```

Comment:
Не забывайте о читаемости кода. Пробелы в выражениях очень важны
```suggestion
    if (num < 2) {
```